internal/pkg/cli/dockerfile.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cli
+
+import (
+	"fmt"
+	"path/filepath"
+	"sort"
+
+	"github.com/spf13/afero"
+)
+
+// listDockerfiles returns the list of Dockerfiles within the current
+// working directory and a sub-directory level below. If an error occurs while
```

Comment:
Is there a specific reason we only walk one sub-dir below cwd instead of recursively walk all of subdirs?

Reply:
I guess it is because of the structure we'd like user to have. Like Root -app1 -app2 -app3, stands for an ecs project with three Dockerfiles in each of the app. Or just Root -Dockerfile standing for an ecs project with only one application. However, if we walk all of subdirs the structure can be a mass and of little value.